documentation/projects/proposals/publish_dataset/20230706-project_proposal.md

```diff
@@ -0,0 +1,207 @@
+  - File format (parquet files? TSVs? a Postgres dump?)
+
+We will also need to coordinate the launch of these efforts and associated
+outreach. See more about that in the [marketing section](#marketing).
+
+## Success
+
+<!-- How do we measure the success of the project? How do we know our ideas worked? -->
+
+This project can be considered a success when the dataset is published. Ideally,
+we will also observe meaningful usage of the dataset. Some ways we might measure
+this include:
+
+- Metrics built into HuggingFace
+  - Models trained with the Dataset are listed
+  - Downloads last month
+  - Likes
+- Our dataset trending on HuggingFace
+- Additionally, we may also see increased interest in our repositories
+- Any positive engagement with our marketing efforts for the project
+
+## Participants and stakeholders
+
+<!-- Who is working on the project and who are the external stakeholders, if any? Consider the lead, implementers, designers, and other stakeholders who have a say in how the project goes. -->
+
+- Openverse maintainers - Responsible for creating the initial raw data dump,
+  maintaining the Openverse account and Dataset on HuggingFace
+- HuggingFace - A key partner responsible for the initial dataset upload,
+  providing advice, and potential marketing collaboration
+- Creative Commons - Stewards of the Commons and CC Licenses, advisors, and
+  another partner in marketing promotion
+- Aaron Ghokaslan - A researcher working on supplementary datasets and providing
+  technical advice
+
+## Infrastructure
+
+<!-- What infrastructural considerations need to be made for this project? If there are none, say so explicitly rather than deleting the section. -->
+
+This project will likely require provisioning some new resources in AWS:
+
+- A dedicated bucket, perhaps a "Requester pays" bucket, for storing the backups
+- New scripts to generate backup artifacts
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this project? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+This project doesn't directly raise any accessibility concerns. However, we
+should be mindful of any changes we would like to make on Openverse.org relating
+to copy edits about this initiative.
+
+We should also be mindful of any accessibility issues with HuggingFace's user
+interface, which we could share with them in an advisory capacity.
+
+## Marketing
+
+<!-- Are there potential marketing opportunities that we'd need to coordinate with the community to accomplish? If there are none, say so explicitly rather than deleting the section. -->
+
+This release will be a big achievement and we should do quite a bit to promote
+it:
+
+- Reach out to past requesters of the dataset and share the HuggingFace link
+- Social channel cross-promotion between the WordPress Marketing team,
```

Comment:
Happy to do cross comms here 🤗

Reply:
Also happy to do cross-comms as well.

Reply:
Great, thank you both! We'll get you in touch with the WordPress project's marketing team in the future to help coordinate.